lib/lure/npm.go

```diff
@@ -35,7 +35,7 @@ func npmOutdated(path string) []moduleVersion {
 	reader := bytes.NewReader(out.Bytes())
 	scanner := bufio.NewScanner(reader)
 
-	npmRegex, _ := regexp.Compile(`([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s*`)
+	npmRegex, _ := regexp.Compile(`([^|\s]+)\s+([^|\s]+)\s+([^|\s]+)\s+([^|\s]+)\s*`)
```

Comment:
what does the output we're trying to parse look like? I don't get any | when I run `npm outdated`

Reply:
wtf are those. Seems like an autocompleted stuff from vscode. Let me investigate